### ISSUE TITLE
Setting the default font: live-set-default-font doesn't seem to be well documented

### ISSUE THREAD

**Issue by Anonymous-Lurker** [overtone/emacs-live#231]
Took me a while to figure out that using the options menu in emacs and setting the default font won't save the font.  Then after customizing my init.el, I could see my font change happen only to be rolled back when the zoning stopped.

More searching brought me to this: https://github.com/overtone/emacs-live/pull/94

I finally added (live-set-default-font "Menlo-18") to my /Users/danm/.live-packs/danm-pack/init.el and found relief.

Can you put this information in an obvious place on the main site or in an FAQ?  I imagine many people change the default font and are frustrated when they have to do it every time they reopen emacs.

Other than that, great job.  I really appreciate what you have done with emacs-live.

**Comment by piotr-yuxuan** [comment]
Yes I change the default font, yes I am every frustrated and burning out of wrath to have to do it every time 😄 Anyway, I first did something like `(set-face-attribute 'default nil :height 120)` but it's always overwritten when the default buffer stops being _live_.

This issue finally gives me what I needed to know but I think elsewhere would be more appropriate.

Other than that, great job. I really appreciate what you have done with emacs-live ^^
